common/lib/xmodule/xmodule/js/spec/sequence/display_spec.js

```diff
@@ -0,0 +1,104 @@
+/* globals Sequence */
+(function() {
+    'use strict';
+
+    describe('Sequence', function() {
+        var keydownHandler,
+            keys = {
+                ENTER: 13,
+                LEFT: 37,
+                UP: 38,
+                RIGHT: 39,
+                DOWN: 40
+            };
+        beforeEach(function() {
+            loadFixtures('sequence.html');
+            window.XBlock = jasmine.createSpyObj('XBlock', ['initializeBlocks']);
+        });
+
+        afterEach(function() {
+            delete window.XBlock;
+        });
+
+        keydownHandler = function(key) {
+            var event = document.createEvent('Event');
+            event.keyCode = key;
+            event.initEvent('keydown', false, false);
+            document.dispatchEvent(event);
```

Comment:
:thumbsup: for this - I was going to suggest using `new Event()` but it turns out it doesn't work on IE, so this is the way to go.